src/cadenza.js

```diff
@@ -327,6 +328,51 @@ export class CadenzaClient {
     return this.#postRequest('setFilter', { filter });
   }
 
+  /**
+   * Set Selection in the currently shown map view.
+   *
+   * @param {WorkbookLayerPath} layer - The data view layer to set the selection in
+   * @param {string[]} values - The variable values
+   * @return {Promise<void>} A 'Promise' for when the selection was set.
+   */
+  setSelection(
+    /** @type WorkbookLayerPath */ layer,
+    /** @type string[] */ values,
+  ) {
+    this.#log('CadenzaClient#setSelection', layer, values);
+    return this.#postRequest('setSelection', { layer, values });
+  }
+
+  /**
+   * Add Selection in the currently shown map view.
+   *
+   * @param {WorkbookLayerPath} layer - The data view layer to add the selection in
+   * @param {string[]} values - The variable values
+   * @return {Promise<void>} A 'Promise' for when the selection was added.
+   */
+  addSelection(
+    /** @type WorkbookLayerPath */ layer,
+    /** @type string[] */ values,
+  ) {
+    this.#log('CadenzaClient#addSelection', layer, values);
+    return this.#postRequest('addSelection', { layer, values });
+  }
+
+  /**
+   * Remove Selection in the currently shown map view.
+   *
+   * @param {WorkbookLayerPath} layer - The data view layer to remove the selection from
+   * @param {string[]} values - The variable values
+   * @return {Promise<void>} A 'Promise' for when the selection was removed.
+   */
+  removeSelection(
+    /** @type WorkbookLayerPath */ layer,
+    /** @type string[] */ values,
+  ) {
+    this.#log('CadenzaClient#removeSelection', layer, values);
+    return this.#postRequest('removeSelection', { layer, values });
```

Comment:
Would it maybe make our live in Cadenza easier if we used only one event with a `mode?: 'add' | 'remove'`? (The API here is fine :+1:)